### DISCUSSION TITLE
创建项目一堆报错，

### DISCUSSION THREAD

**Discussion by imsheng** [NervJS/taro#16553] [Q & A]
1、新创建一个项目使用preact，编译微信小程序后报错 “TypeError: W.createRoot is not a function”
2、clone reactnative 壳工程是旧的0.70的，为什么不直接显示新的？然后切到0.73后yarn upgradePeerdeps又是报错
3、便用reactnative模板集成的，npm install 也是一堆报错，一堆依赖问题.
4、官网的微信群个个都满员也不更新一下
5、4.0都已经出了，文档也不更新一下

**Comment by ShaoGongBra** [comment]
来试一下 duxapp.cn
